sale_exception_holidays_public/tests/test_sale_exception.py

```diff
@@ -22,7 +21,7 @@ def setUpClass(cls):
         # Create holidays
         today = fields.Date.today()
         this_year = today.year
```

Comment:
Maybe you just need to change this value to `holiday_date.year` and you keep the former test logic

Reply:
You're right, I wanted to make a quick fix and didn't see it 😅. Changed